roles/nuage_node/vars/main.yaml

```diff
@@ -0,0 +1,10 @@
+---
+vrs_config: /etc/default/openvswitch
+uplink_interface: "{{ ansible_default_ipv4.interface }}"
+vsp_k8s_yaml: /usr/share/vsp-k8s/vsp-k8s.yaml
+config_dir: "{{ openshift.common.config_base }}/node"
+client_cert: "{{ config_dir }}/system:node:{{ openshift.common.hostname }}.crt"
+client_key: "{{ config_dir }}/system:node:{{ openshift.common.hostname }}.key"
```

Comment:
@detiber @rfredette I think the problem we had in using just the service account was potentially extracting the auth information from the kubeconfig to connect to the API server. The certificate information embedded within the service account's kubeconfig would have to be read?